narwhal/types/src/primary.rs

```diff
@@ -721,7 +730,7 @@ pub struct WorkerReconfigureMessage {
 #[derive(Clone, Debug, Serialize, Deserialize)]
 pub struct WorkerSynchronizeMessage {
     pub digests: Vec<BatchDigest>,
-    pub target: PublicKey,
+    pub target: NetworkPublicKey,
```

Comment:
The only thing i'm not sure of in this PR is changing the type used for `target` here.

Reply:
Yeah I could try to put that back if the `PublicKey` is available where this is called (can't recall off-hand if it is). Originally I was thinking "why not send the key the worker has to use for networking if you have it?" before I realized that it has to translate that from primary key to worker key anyway on the receiving side

Reply:
yeah might just be good to get an opinion from @akichidis on this one.

Reply:
Ok I already put it back but it'd be easy enough for me to undo the commit again :)